xcodeproj/internal/bazel_integration_files/copy_outputs.sh

```diff
@@ -19,6 +19,23 @@ else
       suffix=/Payload
     fi
 
+    if [[ -n ${BAZEL_OUTPUTS_DSYM:-} ]]; then
+      if [[ "$BAZEL_OUTPUTS_DSYM" = *.dSYM ]]; then
+        readonly dsym_files="$BAZEL_OUTPUTS_DSYM"
+        rsync \
+        --copy-links \
+        --recursive \
+        --times \
+        --archive \
+        --delete \
+        ${exclude_list:+--exclude-from="$exclude_list"} \
+        --chmod=u+w \
+        --out-format="%n%L" \
+        $dsym_files \
```

Comment:
This is a space separated list of files, so no quotes

Reply:
But it could have spaces. So you'll need to handle it like `PREVIEW_FRAMEWORK_PATHS` below, where each path is quoted, and use `xargs -n1 sh -c` in order to split it into an array while preserving spaces within the quotes.